examples/kotlin-diktat/warn-dir/chapter1/EnumValueSnakeCaseTest.kt

```diff
@@ -0,0 +1,19 @@
+// ;warn:2:9: [PACKAGE_NAME_INCORRECT_PREFIX] package name should start from company's domain: org.cqfn.save
+package org.cqfn.diktat.test.resources.test.paragraph1.naming.enum_
```

Comment:
Are these resources duplicated in `warn` and `warn-dir`? Why?

Reply:
yes, they have different configuration files, should be working separately